CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## edge-19.1.3
+
+* Controller
+  * Improved API for `ListPods` (thanks @alenkacz!)
```

Comment:
Is there a user-facing impact we can describe for this change?

Reply:
not really; I think this API change was to help us with other controller branches/refactors